fuzzers/libafl-fuzz/src/main.rs

```diff
@@ -80,7 +80,7 @@ fn main() {
                 .output_dir
                 .join(format!("fuzzer_secondary_{}", core_id.0));
             check_autoresume(&fuzzer_dir, &opt.input_dir, opt.auto_resume).unwrap();
-            run_client(state, mgr, &fuzzer_dir, &opt)
+            run_client(state, mgr, &fuzzer_dir, &core_id, &opt)
         })
         .cores(&opt.cores.clone().expect("invariant; should never occur"))
```

Comment:
How do you end up with unparsed cores at this point in time?

Reply:
Since I am using ``clap::skip`` for all env variables, clap automatically runs "default" on the skipped fields. 

This is why I kept it as option and then set it as ``Some``  or error when``AFL_CORES`` is undefined. This is also why it's an invariant that would never occur since at that point, we'd have Cores.

I could not come up with a sensible default for Cores so I kept it as an Option

Reply:
Wouldn't a sensible default be core 0 or something?